tests/parity/toolcalling/table.py

```
@@ -588,7 +588,8 @@ def peer_status(case: dict, dyn: dict, impl: str) -> tuple[str, bool]:
 _TOOL_CALL_MARKUP_RE = re.compile(
     r"</?tool_call|</?tool_calls|<\|tool_call|<\|tool_calls|"
     r"</?TOOLCALL|TOOL_CALLS|<｜(?:DSML｜)?(?:tool|tool▁call|tool▁calls)|"
-    r"<｜DSML｜|</?minimax:tool_call|</?invoke|</?arg_key|</?arg_value"
+    r"<｜DSML｜|</?minimax:tool_call|</?invoke|</?arg_key|</?arg_value|"
```

Ruff (0.15.14)

[warning] String contains ambiguous `｜` (FULLWIDTH VERTICAL LINE). Did you mean `|` (VERTICAL LINE)?

(RUF001)